Bio/Seq.py

```diff
@@ -318,6 +318,18 @@ def defined(self):
         """
         return True
 
+    @property
+    def defined_ranges(self):
+        """Return a tuple of the ranges where the sequence contents is defined.
+
+        The return value has the format ((start1, end1), (start2, end2), ...).
+        """
+        length = len(self)
+        if length > 0:
+            return ((0, length),)
```

Comment:
Assuming CodeCov is right, we should have a test of the "nice case" of a fully defined sequence returning ``((0, length),)``

Reply:
Actually these lines are used only for special cases such as lazy-loaded 2bit files and BioSQL. I added a test for 2bit files.

Reply:
Great - that wasn't clear from the context.